content/fr/docs/concepts/workloads/controllers/statefulset.md

```diff
@@ -0,0 +1,292 @@
+---
+reviewers:
+- enisoc
+- erictune
+- foxish
+- janetkuo
+- kow3ns
+- smarterclayton
```

Comment:
BTW, remember to snip out the list of reviewers; these are English speakers (I assume).